docker/0L/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
```

Comment:
Nit, but would make for a lighter setup: try to go with alpine? IDK if this would be an issue with binaries tho.

Reply:
Good point, thanks.

I'm working on a multi-staged image now. My intuition for using the Easy Mode guide was to be more pragmatic and get something done quick. Now I see that building for source is necessary to correctly run the tower for non-validators.

Let's see whether the built binaries run under alpine. I assume they will have some system deps.

Reply:
If you build them against musl it is possible to get statically linked binaries that will be easily portable. Once you've got static binaries it is possible to have container images that have only the binaries in them and aren't bloated with things like shells and package managers.

Reply:
Thank you @drawks for the input. Do you mind giving a hint on how to build against musl. We could add another make subcommand or simply have that in the Dockerfile. I prefer the earlier not to have building commands spread all over the repo.

Reply:
@nourspace https://blog.davidvassallo.me/2021/06/10/lessons-learned-building-statically-linked-rust-binaries-openssl/  < this article gives a basic example building a static binary with rust and openssl

Reply:
@drawks great, thank you. I will check it this weekend and see where it gets me.